gensim/similarities/annoy.py

```diff
@@ -156,6 +156,7 @@ def build_from_word2vec(self):
 
     def build_from_doc2vec(self):
         """Build an Annoy index using document vectors from a Doc2Vec model."""
+        # FIXME docvecs now = ? What's the replacement for docvecs.count, index_to_doctag, etc?
```

Comment:
`.docvecs` = `.dv` (for compact consistency to its sibling `.wv`)
`.index_to_doctag` = `.index_to_key` (shared class means lookup-value is always 'key')
`dv.count` = `len(dv)` (which worked, & likely would have been clearer, in earlier code too)

Reply:
I figured it out just now, thanks. Will push an update shortly = all migrations complete.